bindings/java/src/blocking_operator.rs

```diff
@@ -24,14 +24,14 @@ use jni::objects::JString;
 use jni::sys::jlong;
 use jni::sys::jstring;
 use jni::JNIEnv;
-
 use opendal::layers::BlockingLayer;
 use opendal::BlockingOperator;
 use opendal::Operator;
 use opendal::Scheme;
 
+use crate::get_global_runtime;
```

Comment:
Why is this style not converged?

Reply:
We are using `imports_granularity = "Item"`